### PR TITLE
Basic Message Storage plugin

### PR DESCRIPTION
**Pull request by usingtechnology**
Signed-off-by: Jason Sherman <tools@usingtechnolo.gy>

### CODE REVIEW

**Review by popkinj — APPROVED** [review]
Appreciate the README

**Comment by loneil** [comment]
Can rebase in from develop to fix the resource strings I believe

File: plugins/basicmessage_storage/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- v0_1: it implements is own version of the `connections/{id}/send-message` endpoint which will persist the sent
+  message. in order to "overwrite" the endpoint, it requires blocking
+  the `aries_cloudagent.aries_cloudagent.protocols.basicmessage` plugin and then registering its messages types,
+- v0_2: uses a middleware wrapper around the existing basicmessage `connections/{id}/send-message` api, and will persist
+  the sent message.
+
+in this case, the `v0_2` pattern is probably best, as we are not changing the signature
+of `connections/{id}/send-message`.
+
```
Comment (Jsyro) [inline plugins/basicmessage_storage/README.md:29]: Is this still true? i only see one set of code within this plugin?

**Review by Jsyro — COMMENTED** [review]
Looks great, just one README comment

File: plugins/basicmessage_storage/README.md (hunk 1 above)
Comment (usingtechnology) [reply plugins/basicmessage_storage/README.md:29]: great catch, that is not true.
will update readme to reflect the code.

**Review by Jsyro — APPROVED** [review]
Code looks great. Nice Job!

**Pull request merged**